src/Tribe/Editor/Attendee_Registration.php

```diff
@@ -115,7 +120,7 @@ public function render() {
 		</style>
 
 		<div id="poststuff"><div class="inside postbox">
-			<a href="<?php echo get_edit_post_link( $this->post->ID, 'raw' );?>">&laquo; Back to Ticket Editor</a>
+			<a href="<?php echo get_edit_post_link( $this->post->ID, 'raw' );?>">&laquo; <?php echo __( 'Back to Editor', 'event-tickets' ) ?></a>
```

Comment:
This should have escaping on the URL and the text

```suggestion
			<a href="<?php echo esc_url( get_edit_post_link( $this->post->ID, 'raw' ) ); ?>">&laquo; <?php esc_html_e( 'Back to Editor', 'event-tickets' ) ?></a>
```